### PR TITLE
create layer advanced [op-34]

### PR DESCRIPTION
**Pull request by Rytiggy**

### CODE REVIEW

File: app/components/add-layer/template.hbs [hunk 1]
```diff
@@ -1,19 +1,25 @@
...
+                    <button class="btn btn-default btn-block" {{action 'addLayer' 'wiki'}}>{{fa-icon "wikipedia-w"}} <br /> Add Wiki layer </button>
                 </div>
-                <div class="col-sm-2">
-                    <button class="btn btn-default btn-block" {{action 'addLayer' 'link'}}>{{fa-icon "link"}} <br /> Link</button>
+                <div class="col-xs-2">
+                    <button class="btn btn-default btn-block" {{action 'addLayer' 'link'}}>{{fa-icon "link"}} <br /> Add Link layer</button>
                 </div>
-                <div class="col-sm-2">
-                    <button class="btn btn-default btn-block" {{action 'addLayer' 'file'}}>{{fa-icon "download"}} <br /> Download</button>
+                <div class="col-xs-2">
```
Comment (caneruguz) [inline app/components/add-layer/template.hbs]: XS-2 will make it tiny in phone screens. Check it out with dev tools but I think it should be sm-2

File: app/components/add-layer/template.hbs [hunk 2]
```diff
@@ -1,19 +1,25 @@
...
-                <div class="col-sm-2">
-                    <button class="btn btn-default btn-block" {{action 'addLayer' 'wiki'}}>{{fa-icon "wikipedia-w"}} <br /> Wiki</button>
+                
+                <div class="col-xs-2">
+                    <button class="btn btn-default btn-block" {{action 'addLayer' 'wiki'}}>{{fa-icon "wikipedia-w"}} <br /> Add Wiki layer </button>
                 </div>
-                <div class="col-sm-2">
-                    <button class="btn btn-default btn-block" {{action 'addLayer' 'link'}}>{{fa-icon "link"}} <br /> Link</button>
+                <div class="col-xs-2">
+                    <button class="btn btn-default btn-block" {{action 'addLayer' 'link'}}>{{fa-icon "link"}} <br /> Add Link layer</button>
```
Comment (caneruguz) [inline app/components/add-layer/template.hbs]: Do we need Add and layer in the language? I removed them because I thought they were not strictly necessary. If we keep 'add' then 'layer' doesn't need to be there. It's more of our internal name for it anyway.

File: app/components/add-layer/template.hbs [hunk 3]
```diff
@@ -1,19 +1,25 @@
...
+                <div class="col-xs-2">
+                    <button class="btn btn-default btn-block" {{action 'addLayer' 'file'}}>{{fa-icon "download"}} <br /> Add Download layer</button>
                 </div>
+
                 <div class="col-sm-2">
                     <button class="btn btn-default btn-block" {{action 'addLayer' 'image'}}>{{fa-icon "image"}} <br /> Background Image</button>
                 </div>
+                <div class="col-xs-2">
+                    <button class="btn btn-default btn-block" {{action 'addLayer' 'advanced'}}>{{fa-icon "paint-brush"}} <br /> Add Advanced layer</button>
+                </div>
```
Comment (caneruguz) [inline app/components/add-layer/template.hbs]: 'Advanced' layer is fine for internal use but I think we should brainstorm another name for it in the UI

File: app/components/layer-advanced/component.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+
+  didRender() {
+    this._super(...arguments); 
+    this.$('div#froala-editor').html(this.get('layer.content'));
+    if(this.get("editMode") && this.get('showSettings') == false){
```
Comment (caneruguz) [inline app/components/layer-advanced/component.js]: Does this work too? ```!this.get('showSettings')``` instead of ```this.get('showSettings') == false)```

Also if you will use the == check you should use the strict check === 
https://github.com/airbnb/javascript#comparison-operators--equality

File: app/components/layer-advanced/style.scss [hunk 5]
```diff
@@ -0,0 +1,3 @@
+.layer-footer{
+    background: #f8f8f8;
```
Comment (caneruguz) [inline app/components/layer-advanced/style.scss]: What is this style for?

File: app/components/layer-advanced/template.hbs [hunk 6]
```diff
@@ -0,0 +1,4 @@
+{{yield}}
```
Comment (caneruguz) [inline app/components/layer-advanced/template.hbs]: You can remove the yields, I don't think we will inject anything into these components.

File: app/components/layer-wrapper/template.hbs [hunk 7]
```diff
@@ -4,8 +4,10 @@
             <div class="col-xs-12">
                 <div class="layer clearfix">
                     <div class="layer-content">
-
-                        {{component layer.component layer=layer node=node layers=layers editMode=editMode theme=theme}}
+                        {{#if editMode}}
```
Comment (caneruguz) [inline app/components/layer-wrapper/template.hbs]: Why did you move this back up?

File: app/components/add-layer/template.hbs (hunk 3 above)
Comment (Rytiggy) [inline app/components/add-layer/template.hbs]: Custom layer?

File: app/components/layer-advanced/style.scss (hunk 5 above)
Comment (Rytiggy) [inline app/components/layer-advanced/style.scss]: removed

File: app/components/add-layer/template.hbs (hunk 3 above)
Comment (caneruguz) [inline app/components/add-layer/template.hbs]: We can come back to this later
